src/components/TopBar/TopBarPageTitle.js

```diff
@@ -14,10 +14,7 @@ export const TopBarPageTitle = ({ children, ...props }) => {
 
   return (
     <div className={classes.topBarPageTitle}>
-      <Typography
-        variant="h6"
-        style={{ color: 'rgba(255, 255, 255, 0.87)' }}
-        {...props}>
+      <Typography variant="h6" style={{ color: colors.white87 }} {...props}>
```

Comment:
I think we could add this style to [theme.js](https://github.com/DarthVanger/NeuralNotes/blob/master/src/theme.js#L16), where we have all the other styles for `h6`.

Reply:
Add ```typography: title1``` style to theme.js